style.css

```diff
@@ -256,51 +251,43 @@ a {
 }
 
```

Comment:
- [ ] Kindly fix work section, since there's no margin on this size

![image](https://user-images.githubusercontent.com/93630700/214734788-101a924c-4482-4a82-8654-fc61e4eb5dab.png)